buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -302,9 +295,11 @@ private static void autoConfigureClusterDependencies(
         project.afterEvaluate(ip -> container.forEach(esNode -> {
             // declare dependencies against artifacts needed by cluster formation.
             String dependency = String.format(
-                "org.elasticsearch.distribution.zip:%s:%s@zip",
-                esNode.getDistribution().getFileName(),
-                esNode.getVersion()
+                "unused:%s:%s:%s@%s",
```

Comment:
What is "unused"?

Reply:
the group name, not used right now since the repo is a flat dir. 
Of course this will only work in tests right now, but I plan to deal with this as a follow up as I prepare changes to use testclusters with rest integ test task.